torch/testing/_internal/common_methods_invocations.py

```diff
@@ -133,15 +135,22 @@ def should_skip(self, cls_name, test_name, device_type, dtype):
 
         return False
 
-    def supports_dtype(self, dtype, device_type):
+    def supported_dtypes(self, device_type):
         if device_type == 'cpu':
-            return dtype in self.dtypesIfCPU
+            return self.dtypesIfCPU
         if device_type == 'cuda':
-            if TEST_WITH_ROCM:
-                return dtype in self.dtypesIfROCM
-            return dtype in self.dtypesIfCUDA
+            return self.dtypesIfROCM if TEST_WITH_ROCM else self.dtypesIfCUDA
+        else:
+            return self.dtypes
+
+
+    def supports_dtype(self, dtype, device_type):
+        return dtype in self.supported_dtypes(device_type)
 
-        return dtype in self.dtypes
+    def tested_dtypes(self, device_type):
```

Comment:
Name would be updated here, too. 

Add a comment here elaborating on what this function does, and then point people to the enum, which can have a more descriptive comment about the thinking behind each option, how they're intended to be used, and what the default behavior is.